### PR TITLE
switch from pure python to pytest for integration tests

### PR DESCRIPTION
**Pull request by woglinde**

### CODE REVIEW

File: integration-tests/.dockerignore [hunk 1]
```diff
@@ -1,3 +1,9 @@
 Dockerfile
+.dockerignore
```
Comment (zecke) [inline integration-tests/.dockerignore]: Why is this one needed?

File: integration-tests/Dockerfile [hunk 2]
```diff
@@ -1,2 +1,7 @@
-FROM python:2-onbuild
-CMD [ "python", "./osmo-smsc-integrationtests.py", "-H", "osmo-smsc-om"]
+FROM python:2-alpine
```
Comment (zecke) [inline integration-tests/Dockerfile]: What is the difference between onbuild and alpine?

Comment (woglinde) [inline integration-tests/Dockerfile]: Main reason to switch is, that the alpine image uses only 80mb of space, where the onbuild uses 400 or more. So it saves us some downloadtime. The other diffrence is that the on-build has already the pip install stanca.

File: integration-tests/.dockerignore (hunk 1 above)
Comment (woglinde) [inline integration-tests/.dockerignore]: That was for build speed up when the integrationtest was copied inside the dockerimage. With the current approach to runtime mount the directory with the test. This file is obsolte I wll delete it.

**Pull request merged**